project/scripts/release

```diff
@@ -272,6 +272,10 @@ else
   RELEASE_OPT="-Dakka.genjavadoc.enabled=true"
 fi
 
+# tag this release
+echolog "Tagging..."
+try git tag -am "Version ${version}" v${version}
```

Comment:
does it cleanup the tag if things fail? maybe already taken care of by the bail out?

Reply:
yes, it's removed in `git_cleanup` which is called from `bail_out`. `try` makes sure `bail_out` is called when a command fails.

After pushing, the error handling no longer calls `bail_out`, but logs a message that you'll have to manually clean up - this hasn't changed in this PR.